kornia/feature/laf.py

```diff
@@ -51,18 +52,18 @@ def get_laf_center(LAF: torch.Tensor) -> torch.Tensor:
         >>> output = get_laf_center(input)  # BxNx2
     """
     KORNIA_CHECK_LAF(LAF)
-    out: torch.Tensor = LAF[..., 2]
+    out: Tensor = LAF[..., 2]
     return out
 
 
-def get_laf_orientation(LAF: torch.Tensor) -> torch.Tensor:
+def get_laf_orientation(LAF: Tensor) -> Tensor:
     """Return orientation of the LAFs, in degrees.
 
     Args:
-        LAF: (torch.Tensor): tensor [BxNx2x3].
+        LAF: (Tensor): tensor [BxNx2x3].
```

Comment:
```suggestion
        LAF: tensor [BxNx2x3].
```
Types are inferred automatically in docs